nautobot/dcim/templates/dcim/device/lldp_neighbors.html

```diff
@@ -27,7 +27,7 @@
                             <td class="configured_device" data="{{ iface.connected_endpoint.device }}" data-chassis="{{ iface.connected_endpoint.device.virtual_chassis.name }}">
                                 {{ iface.connected_endpoint.device|hyperlinked_object }}
                             </td>
-                            <td class="configured_interface" data="{{ iface.connected_endpoint }}">
+                            <td class="configured_interface" data-interface-name="{{ iface.connected_endpoint }}" data-mac-address="{{ iface.connected_endpoint.mac_address }}">
```

Comment:
Actually, I wonder - should we actually render the configured MAC address as an additional column in the table instead?

Reply:
I'm happy to add this if you like.

Reply:
I'm thinking based on the screenshots that it would makes sense just as a way to show how the "match" was determined for a given interface. Up to you.

Reply:
I like the idea. Just updated with this:

![Screen Shot 2024-03-15 at 10 31 05 AM](https://github.com/nautobot/nautobot/assets/358901/bc7a6095-6c39-47d6-9c80-9efbd5c0119e)